common/lib/capa/capa/responsetypes.py

```diff
@@ -364,14 +369,22 @@ def make_hint_div(self, hint_node, correct, student_answer, question_tag,
 
         # Form the div-wrapped hint texts
         hints_wrap = u''.join(
-            [u'<div class="{0}">{1}</div>'.format(QUESTION_HINT_TEXT_STYLE, dct.get('text'))
-             for dct in hint_log]
+            [u'<div class="{question_hint_text_style}">{text}</div>'.format(
```

Comment:
You will need to use HTML and Text on these chunks of HTML to properly escape them. It will be necessary to determine if dct.get('text') is always plain text (in which case it should be wrapped with the Text function).

I'd recommend you run the safecommit linter locally, as it often provides useful suggestions on how to fix issues.